### PR TITLE
HARVESTER: Implement pod affinity in harvester manager

### PR DESCRIPTION
**Pull request by DaiYuzeng**
### Occurred changes and/or fixed issues

- https://github.com/harvester/harvester/issues/2317

### Screenshot/Video

image

### CODE REVIEW

**Comment by WuJun2016** [comment]
Hi,  @richard-cox   @mantis-toboggan-md  Will this PR be merged into Rancher 2.7.3 ?

File: shell/components/form/PodAffinity.vue [hunk 1]
```diff
@@ -125,6 +133,14 @@ export default {
     hasNamespaces() {
       return this.allNamespaces.length;
     },
+
+    namespaceModeLabel() {
```
Comment (DaiYuzeng) [inline shell/components/form/PodAffinity.vue:137]: Hi @richard-cox ! Please be aware to the changes of `PodAffinity`, I'm not sure this is a good inspiration. But I have no idea that how can I change the labels when harvester using the `PodAffinity`. If you have any ideas, please let me know, thank you very much! cc @WuJun2016

**Comment by DaiYuzeng** [comment]
According to the #8643 changes, I propose that harvester should clone this component to the pkg, and revert the changes for harvester. I think we should separate the similar component timely. @WuJun2016 

I am going to send PR to revert the changes for harvester.

**Comment by DaiYuzeng** [comment]
Hi @mantis-toboggan-md ! Could you take a look at this PR?

**Pull request merged**